jwala-tomcat/src/main/resources/data/scripts/linux/jvm-service.sh

```diff
@@ -12,115 +12,27 @@
 ### END INIT INFO
 
 # Source LSB function library.
-. /etc/init.d/functions
 
-NAME=`basename "$0"`
+[Unit]
+Description= jwala service file
+After=network.target
 
-TOMCAT_HOME=@TOMCAT_HOME@
-TOMCAT_HOME_BIN=$TOMCAT_HOME/bin
+[Service]
+Type=forking
 
 # Define the tomcat username
-TOMCAT_USER="${TOMCAT_USER:-tomcat}"
+User=tomcat
 
-# Define the tomcat group
-TOMCAT_GROUP="${TOMCAT_GROUP:-`id -gn $TOMCAT_USER`}"
+#Environment="JAVA_HOME=/usr/lib/jvm/jre"
+#Environment="JAVA_OPTS=-Djava.security.egd=file:///dev/urandom"
 
-CATALINA_PID=$TOMCAT_HOME/logs/catalina.pid
-export CATALINA_PID
+Environment="CATALINA_BASE=@TOMCAT_HOME@"
+Environment="CATALINA_HOME=@TOMCAT_HOME@"
+Environment="CATALINA_PID=@TOMCAT_HOME@/logs/catalina.pid"
+Environment="CATALINA_HOME_BIN=@TOMCAT_HOME@/bin"
 
-RETVAL="0"
+ExecStart=/opt/ctp/jwala-0.0.245/apache-tomcat-7.0.55/bin/startup.sh
```

Comment:
These paths can't be hardcoded.
See the install-service.sh for how tokens were replaced in the previous version of the script.